.github/workflows/datadog-static-analysis.yml

```diff
@@ -0,0 +1,21 @@
+on: [push]
+
+name: Datadog Static Analysis
+
+jobs:
+  static-analysis:
+    runs-on: ubuntu-latest
+    name: Datadog Static Analyzer
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Check code meets quality and security standards
+      id: datadog-static-analysis
+      uses: DataDog/datadog-static-analyzer-github-action@v1
```

Comment:
Sorry, I'm unfamiliar with this, so possibly dumb question. I was reading the docs for DD static analysis and saw:
> Static Analysis takes the guesswork out of adhering to an organization’s code standards, enabling your development team to ship compliant code without significant impacts to developer velocity.

Do we define these rules somewhere? Just want to know what rules we'll be applying to the repo so we can fix any issues before merging this (otherwise I'm worried it'll block our PRs or become an un-required check that gets ignored)

Reply:
The rules used are defined in the `static-analysis.datadog.yml` file, defined at the root of the repository. For this PR, we only add rules requested by security but more rules are available per [our documentation](https://docs.datadoghq.com/code_analysis/static_analysis_rules/).

Reply:
Sorry to clarify, I meant where are rulesets like `sit-ci-best-practices` defined? I did a quick search in GitHub and the docs but I just see them being called in other repos `static-analysis.datadog.yml` files

Reply:
For visibility: clarified on Slack since this ruleset is not documented now.